### DISCUSSION TITLE
In File search for PDF?

### DISCUSSION THREAD

**Discussion by Heneeth** [scambier/obsidian-omnisearch#351] [Q&A]
Is there a way to search inside file? It works for "md" file in omnisearch. But when I try to hit tab and attempt to search in file for PDF, it doesn't. I have a M1 Mac.

**Comment by scambier** [comment]
Hi! No, sorry, Omnisearch doesn't support in-file search for documents that go through Text Extractor. There's this open issue https://github.com/scambier/obsidian-omnisearch/issues/100 but it's unlikely I ever fix this 🤷‍♂️
